lib/config-generator.js

```diff
@@ -166,7 +166,7 @@ const compat = new FlatCompat({baseDirectory: __dirname, recommendedConfig: plug
         }
 
         if (this.answers.language === "typescript") {
-            this.result.devDependencies.push("typescript-eslint");
+            this.result.devDependencies.push("typescript-eslint@rc-v8");
```

Comment:
I'm not sure if we should be installing prerelease versions. Maybe rather wait a bit for the final v8.0.0?

Reply:
the problem: eslint v9 + tseslint(latest) could cause an installing error.

I also prepared a fix to use `npm install --force` to fix it: https://github.com/eslint/create-config/pull/118. Which one do you prefer?

Reply:
/cc @nzakas @fasttime

Reply:
I'd use `--force` for now.